tests/api/api.spec.ts

```diff
@@ -497,13 +497,11 @@ describe("@dataform/api", () => {
     it("schema overrides", async () => {
       const graph = await compile({
         projectDir: path.resolve("df/examples/bigquery"),
-        defaultSchemaOverride: "overridden_default_schema",
-        assertionSchemaOverride: "overridden_assertion_schema"
+        schemaSuffix: "suffix",
       });
-      expect(graph.projectConfig.defaultSchema).to.equal("overridden_default_schema");
-      expect(graph.projectConfig.assertionSchema).to.equal("overridden_assertion_schema");
+      expect(graph.projectConfig.schemaSuffix).to.equal("suffix");
       graph.tables.forEach(table =>
-        expect(table.target.schema).to.equal("overridden_default_schema")
+        expect(table.target.schema).to.match(/.*_suffix$/)
```

Comment:
match the whole string plz? (that way we test the appending works as we expect it to)

Reply:
Bit fiddly now as we have one overridden schema, but this basically does the trick..